discovery-provider/src/models.py

```diff
@@ -879,3 +879,28 @@ def __repr__(self):
 txhash={self.txhash},\
 created_at={self.created_at},\
 updated_at={self.updated_at})>"
+
+class TrackRoute(Base):
+    __tablename__ = "track_routes"
+
+    slug = Column(String, nullable=False)
```

Comment:
Some documentation here esp re: the difference between slug & title_slug would be great